From hackermail Thu Jan  1 00:00:00 1970
From: SeongJae Park <sj@kernel.org>
To: Andrew Morton <akpm@linux-foundation.org>
Cc: SeongJae Park <sj@kernel.org>
Cc: damon@lists.linux.dev
Cc: kernel-team@meta.com
Cc: linux-kernel@vger.kernel.org
Cc: linux-mm@kvack.org
Date: Tue, 15 Jul 2025 10:12:09 -0700
Subject: [PATCH mm-unstable 2/2] mm/damon: remove damon_ctx->callback kernel-doc
         comment

Commit 769bd386e490 ("mm/damon/core: add cleanup_target() ops callback")
on mm-unstable tree removed callback field of damon_ctx, but not erased
the kernel-doc comment for the field.  This results in build warning.
Fix it by removing the obsolete comment.

Fixes: 769bd386e490 ("mm/damon/core: add cleanup_target() ops callback")
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Fixes: 769bd386e490 ("mm/damon/core: add cleanup_target() ops callback") # mm-unstable
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 include/linux/damon.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/linux/damon.h b/include/linux/damon.h
index efabc3ba52ee..f13664c62ddd 100644
--- a/include/linux/damon.h
+++ b/include/linux/damon.h
@@ -746,7 +746,6 @@ struct damon_attrs {
  * Accesses to other fields must be protected by themselves.
  *
  * @ops:	Set of monitoring operations for given use cases.
- * @callback:	Set of callbacks for monitoring events notifications.
  *
  * @adaptive_targets:	Head of monitoring targets (&damon_target) list.
  * @schemes:		Head of schemes (&damos) list.
-- 
2.39.5
